include/dfu/mcuboot.h

```diff
@@ -17,26 +17,57 @@
 extern "C" {
 #endif
 
+#ifdef BOOT_SWAP_TYPE_NONE
+#if BOOT_SWAP_TYPE_NONE != 1 /*ensure the same definition in MCUBoot */
```

Comment:
The proper case would be `MCUboot` :-)

Reply:
Do you mean BOOT_SWAP_TYPE_NONE -> MCUBOOT_SWAP_TYPE_NONE.
This will impose on users (mcumgr, updatehub...), that why I keep naming.

Reply:
Hehe, no I mean in the comment you should change "MCUBoot" to "MCUboot"